<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FryPlotter</class>
 <widget class="QDialog" name="FryPlotter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FryPlotter</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>600</y>
     <width>161</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Close|QDialogButtonBox::Help</set>
   </property>
  </widget>
  <widget class="QComboBox" name="targetLayer">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Data Layer</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Fry Analysis</string>
   </property>
  </widget>
  <widget class="QPushButton" name="calcButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>610</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>20</y>
     <width>531</width>
     <height>611</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="FryPlot">
    <attribute name="title">
     <string>Fry Plot</string>
    </attribute>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>20</y>
       <width>511</width>
       <height>441</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="fryplotLayout"/>
    </widget>
    <widget class="QWidget" name="fryplotToolbar" native="true">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>490</y>
       <width>431</width>
       <height>80</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="RoseDiagram">
    <attribute name="title">
     <string>Rose Diagram</string>
    </attribute>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>9</y>
       <width>501</width>
       <height>471</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="roseLayout"/>
    </widget>
    <widget class="QWidget" name="roseToolbar" native="true">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>500</y>
       <width>481</width>
       <height>80</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="FrylengthHist">
    <attribute name="title">
     <string>Fry Length Histogram</string>
    </attribute>
    <widget class="QWidget" name="verticalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>501</width>
       <height>471</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="frylenLayout"/>
    </widget>
    <widget class="QWidget" name="frylenToolbar" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>500</y>
       <width>481</width>
       <height>80</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>171</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Outputs</string>
   </property>
   <widget class="QCheckBox" name="chkExTxt">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>161</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Output to Text File</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkPseuPlot">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Psuedo-located Fry Plot</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkFSPlot">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Fry-space Plot</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <property name="tristate">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkRsDiag">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Rose Diagram</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkFVPlot">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>181</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Fry vector length histogram</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>231</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Pseudo Plot Settings</string>
   </property>
   <widget class="QLineEdit" name="openFpseudo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="openFbtn">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>20</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkPseLoadcan">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Load to Canvas</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <zorder>groupBox</zorder>
  <zorder>buttonBox</zorder>
  <zorder>targetLayer</zorder>
  <zorder>label</zorder>
  <zorder>label_3</zorder>
  <zorder>calcButton</zorder>
  <zorder>tabWidget</zorder>
  <zorder>groupBox_2</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>FryPlotter</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>FryPlotter</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>helpRequested()</signal>
   <receiver>FryPlotter</receiver>
   <slot>open()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>200</x>
     <y>255</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkPseuPlot</sender>
   <signal>toggled(bool)</signal>
   <receiver>groupBox_2</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>90</x>
     <y>128</y>
    </hint>
    <hint type="destinationlabel">
     <x>125</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
